Backend/api/workout/model.js

```diff
@@ -49,5 +48,7 @@ const workoutSchema = new mongoose.Schema({
   },
 });
 
+workoutSchema.index({ date: 1 }, { expireAfterSeconds: 0 });
```

Comment:
I think this deleted the workout exactly in the day of the workout.

I think expireAfterSeconds should be set to 24 * 60 * 60 seconds. We should discuss this when we meet.